core/org/commcare/core/parse/UserXmlParser.java

```diff
@@ -82,10 +78,16 @@ public User parse() throws InvalidStructureException, IOException, XmlPullParser
             }
         }
 
+        addCustomData(u);
+
         commit(u);
         return u;
     }
 
+    public void addCustomData(User u){
+        //Don't do anything in base class
```

Comment:
Oh, ha! I see. I was thinking the Product version of "custom user data", which is why I was confused. This makes sense, thanks.